src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/tdd/complex-process/logic/App.cs

```diff
@@ -76,9 +76,14 @@ public override async Task<bool> RunAppEvent(AppEventType appEvent, object model
         /// </summary>
         /// <param name="validationResults">Object to contain any validation errors/warnings</param>
         /// <returns>Value indicating if the form is valid or not</returns>
-        public override async Task RunValidation(object data, ModelStateDictionary validationResults)
+        public override async Task RunDataValidation(object data, ModelStateDictionary validationResults)
         {
-            await _validationHandler.Validate(data, validationResults);
+            await _validationHandler.ValidateData(data, validationResults);
+        }
+
+        public override async Task RunTaskValidation(Instance instance, string taskId, ModelStateDictionary validationResults)
```

Comment:
'ModelStateDictionary validationResults' dette er variabelen jeg ønsker å bytte ut med List<ValidationIssue> . Kallet til denne skjer fra ValidationAppSI og mappes til ValidationIssue før det returneres tilbake til controlleren. Men ser vi for oss at utvikler vil bruke denne typen andre steder i løsningen?